### DISCUSSION TITLE
Memory consumption

### DISCUSSION THREAD

**Discussion by sregg** [getsentry/sentry-react-native#2464] [Ideas]
Is there a plan to track memory consumption in mobile apps (more specifically react-native) as part of Performance Monitoring?

Datadog RUM offers that feature:

<img width="1125" alt="image" src="https://user-images.githubusercontent.com/1479283/189401224-658b5d67-3c46-4709-87f0-35684132b699.png">

**Comment by marandaneto** [comment]
@sregg we're currently working on something similar for Android and iOS, See https://github.com/getsentry/team-mobile/issues/25
After doing it on the Native SDKs, we intend to expand to React Native and Flutter.
